zerotier/files/etc/init.d/zerotier

```diff
@@ -47,7 +47,7 @@ start_service() {
 	rm -rf "${CONFIG_PATH}"
 
 	# Create link or copy files from config_path to CONFIG_PATH
-	if [ -n "${config_path}" -a "${config_path}" != "${path}" ]; then
```

Comment:
@mwarning this comparison is no longer needed, I forgot to remove it when I submitted the changes in #120